vars/ppgOperatingSystems.groovy

```diff
@@ -1,3 +1,3 @@
 def call() {
-  return ['centos-7', 'debian-9', 'debian-10', 'ubuntu-focal', 'ubuntu-bionic', 'rhel8']
+  return ['centos-7', 'centos8', 'debian-9', 'debian-10', 'ubuntu-focal', 'ubuntu-bionic', 'rhel8']
```

Comment:
Are you sure that we need to run the tests on  centos8 and rhel8?

Reply:
Yes, we need that for different minor versions (8.2 and 8.3).

Reply:
Are you sure that we need to test 8.2? @EvgeniyPatlan  what do you think ?

Reply:
@mu-samoylov we need to perform tests always on latest available OS
otherwise we would need to add all OS from 7.0 to 7.9 and from 8.0 to 8.3

Reply:
@EvgeniyPatlan are you sure that it's ok to say to customers update your distro to latest version? I don't think so. At current moment I prefer to have a both versions. Testing is in my area of responsibility, not yours. So please don't argue, but accept the pull request.

Reply:
yes

Reply:
at least please add comment which would describe what version for which image is used. Otherwise this looks very strange

Reply:
Comment added.